distributor-node/src/services/networking/NetworkingService.ts

```diff
@@ -437,16 +447,16 @@ export class NetworkingService {
   }
 
   async getQueryNodeStatus(): Promise<StatusResponse['queryNodeStatus']> {
-    const qnState = await this.queryNodeApi.getQueryNodeState()
+    const squidStatus = await this.queryNodeApi.getQueryNodeState()
```

Comment:
I think we should try-catch this? Relating to https://github.com/Joystream/joystream/pull/5001/files#r1443286342